crates/ruff_python_formatter/src/preview.rs

```diff
@@ -44,3 +44,12 @@ pub(crate) fn is_empty_parameters_no_unnecessary_parentheses_around_return_value
 pub(crate) fn is_match_case_parentheses_enabled(context: &PyFormatContext) -> bool {
     context.is_preview()
 }
+
+/// This preview style fixes a bug with the docstring's `line-length` calculation when using the `dynamic` mode.
+/// The new style now respects the indent **inside** the docstring and reduces the `line-length` accordingly
+/// so that the docstring's code block fits into the global line-length setting.
+pub(crate) fn is_docstring_code_block_in_docstring_indent_enabled(
+    context: &PyFormatContext,
+) -> bool {
+    context.is_preview()
+}
```

Comment:
How come this is treated as a new style instead of a bug fix? (Sorry, I'm probably out of step with what our policy is here.)

Reply:
The formatter versioning policy is rather strict:

> The stable style changed to prevent invalid syntax, changes to the program's semantics, or removal of comments

It only allows hard correctness errors. The only other permitted changes are bug fixes that don't change any already formatted code. For example, we can fix a bug where the formatter incorrectly collapses two blank lines to one instead of preserving one or two. 

The motivation for this strict rule is that a `ruff format --check` command shouldn't start failing between patch versions for already formatted code.